composer/models/gpt2/scaling_laws_generator.py

```diff
@@ -177,10 +177,7 @@ def parse_args():
     'eval_batch_size': 8,
     'seed': 17,
     'accelerator': {
-        'gpu': {
-            'n_gpus': 1,
```

Comment:
Woops, looks like I missed this. Thanks for catching!

Reply:
(This code doesn't have test cases :D )